README.md

```diff
@@ -6,18 +6,22 @@ Table of Contents
 * [Overview](#Overview)
 * [Setup](#Setup)
 * [How to Run Tests](#How-to-Run-Tests)
-* [Test Files](#Test-Files)
+* [TPCH Tests](#TPCH-Tests)
   * [Test Case Args](#Test-Case-Args)
   * [Substrait Plans](#Substrait-Plans)
   * [SQL Queries](#SQL-Queries)
-* [How to Add New Consumers](#How-to-Add-New-Consumers)
+* [Substrait Function Tests](#Substrait-Function-Tests)
```

Comment:
Minor nit: Maybe just "function tests" or "extension function tests"?  The "Substrait" part is probably implied.